config/jobs/image-pushing/k8s-staging-cloud-provider-aws.yaml

```diff
@@ -0,0 +1,31 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated for each repo.
+  kubernetes/cloud-provider-aws:
+    - name: cloud-provider-aws-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-aws-cloud-provider-aws-image-pushes
```

Comment:
also add to testgrid? e.g. https://github.com/kubernetes/test-infra/pull/18511/files

Reply:
Added to PR